src/styles/index.scss

```diff
@@ -1,3 +1,129 @@
-body {
+.page {
   margin: 0;
+  box-sizing: border-box;
+  width: 200px;
+  background: rgba(255, 255, 255, 1);
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+}
+
+.card__img {
+  height: 134px;
+  width: 160px;
+
+  background-position: center;
+  background-size: cover;
+  margin: 33 20 0;
+}
+
+.card__title {
+  display: block;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin: 40 17 0;
+  color: rgba(6, 11, 53, 1);
+  height: 36px;
+}
+
+.card__code {
+  display: block;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: rgba(97, 96, 112, 1);
+  margin: 4 0 0 17;
+  height: 14px;
+}
+
+.card__review {
+  display: flex;
+  width: 166px;
+  justify-content: space-between;
+  margin: 17 17 0;
+  align-items: center;
+  height: 16px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star-active.svg);
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:not(:first-child) {
+  margin-left: 4px;
+}
+
+.stars__star:nth-child(5) {
+  background-image: url(../images/star.svg);
+}
+
+.card__review-amount {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  text-align: right;
+  color: rgba(6, 11, 53, 1);
+  margin-left: 16px;
+}
+
+.card__price {
+  display: flex;
+  margin: 24 17 0;
+  justify-content: space-between;
+  align-items: center;
+  width: 166px;
+  height: 18px;
+}
+
+.card__price-title {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  color: rgba(97, 96, 112, 1);
+}
+
+.card__price-amount {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  color: rgba(6, 11, 53, 1);
+}
+
+.card__buy {
+  display: flex;
+  width: 166px;
+  height: 40px;
+  margin: 16px;
```

Comment:
The margin property in `.card__buy` should be `margin: 16px;` to include the units.